src/sentry/integrations/vsts/integration.py

```diff
@@ -221,8 +221,16 @@ def build_integration(self, state):
 
     def create_subscription(self, instance, account_id, oauth_data):
         webhook = WorkItemWebhook()
-        subscription, shared_secret = webhook.create_subscription(
-            instance, oauth_data, self.oauth_redirect_url, account_id)
+        try:
+            subscription, shared_secret = webhook.create_subscription(
+                instance, oauth_data, self.oauth_redirect_url, account_id)
+        except ApiError:
+            # TODO(lb): how do I show an error to the user?
+            # TODO(lb): how do I make sure this error is the right one? Should find a
+            # way to ensure that.
+            raise ApiError(
+                'Your permissions are not sufficient to create an integration. Please check with the owner of this account.')
```

Comment:
I'm not sure `APIError` is going to be what we want here. The Pipeline stuff seems more like the `web/frontend/...` controllers/views than API endpoints.

This is probably a question for @EvanPurkhiser, but maybe we should be using `django.contrib.messages` here?

Reply:
We can probably render an error modal. I _think_ I did this somewhere, but I'm trying to remember where..

Reply:
@EvanPurkhiser this is currently done in Django forms. I can create an error modal if it's just a matter of adding another view with Django. Or I could even loop so that we go back to the Account selection with an error message written somewhere. But the moment we hit react is the moment I'm going to need assistance. So let me know what the desired behavior is.

Reply:
https://github.com/getsentry/sentry/blob/44f78ad181dba98b280e4406a743790f588b2cf1/src/sentry/pipeline/__init__.py#L218-L220 Use this :)